index.html

```diff
@@ -0,0 +1,162 @@
+            </p>
+            </div>
+            
+            <div class="action">
+               <h3>Let's get you started right</h3>
+               <button>Click here to begin!</button>
+            </div>
+           
+          </div>
+          <div class="programs">
+            <h2>Courses</h2>
+            <hr />
+            <div class="courses"><div class="program-card">
+              <img src="./img/web-design.png" alt="frontend development" />
+              <h2>Frontend engineering</h2>
+              <p>Learn the use programming languages such as HTML, CSS, and JavaScript to create and style web pages, add interactivity and functionality, and ensure a seamless user experience. Build website and applications that are visually appealing, user-friendly, and accessible.</p>
+            </div>
+            <div class="program-card">
+              <img src="./img/backend.png" alt="backend development"/>
+              <h2>Backend engineering</h2>
+              <p>Learn Python, Ruby, Java, and PHP to develop server-side code, APIs, and databases. Create and maintain the infrastructure and systems that handle data processing, storage, and retrieval, as well as handling requests from the frontend and delivering responses.</p>
+            </div>
+            <div class="program-card">
+              <img src="./img/product.png" alt="Product design"/>
+              <h2>Product design</h2>
+              <p>Learn the process of conceptualizing, creating, and refining physical or digital product to meet the needs and expectations of its intended users. Learn figma</p>
+            </div>
+            <div class="program-card">
+              <img src="./img/data.png" alt="data analytics"/>
+              <h2>Data Analytics</h2>
+              <p>Learn and master tools and techniques to analyze data, including data mining, machine learning, and predictive analytics. Learn to build data sytems with python</p>
+            </div>
+              <div class="program-card">
+              <img src="./img/project.png" alt="project management"/>
+              <h2>Project management</h2>
+              <p>Learn the practice of planning, organizing, and overseeing the completion of a project from start to finish. Learn how to coordinate resources, tasks, and activities required to achieve project goals within a specified timeframe and budget.</p>
+            </div></div>
+            
+            <button>Find something interesting? Join now</button>
+            <a href="#" class="view-list">View our full list of courses</a>
+          </div>
+
+          <div class="instructors">
+            <h2>
+              Our Instructors
+            </h2>
+            <hr/>
+            <div class="profiles">
+            </div>
+          <button class="btn-more">More <img src="./img/arrow_down 1.png" alt="show more arrow down"/></button>
+          </div>
+
+          <div class="partners">
+                <h1>Partners</h1>
```

Comment:
- [ ] As suggested by the previous reviewer, There should be only one `h1` in an HTML file because of semantics and it helps search engines and web crawlers easily understand the main subject of your page.

Reply:
Done, thanks